CMakeLists.txt

```
@@ -155,3 +155,14 @@ if (PYBIND11_INSTALL)
     endif()
   endif()
 endif()
+
+# uninstall target
+if(NOT TARGET uninstall)
```

```suggestion
if(PYBIND11_MASTER_PROJECT)
```

(you actually don't need to check for the target existing in this case, since master project doesn't have this target)

---

Thanks for reviewing @henryiii . This is a good suggestion, didn't even think about it. I'll commit it in.